src/neptune/common/backends/utils.py

```diff
@@ -45,14 +52,42 @@
     NeptuneInvalidApiTokenException,
     NeptuneSSLVerificationError,
     Unauthorized,
+    WritingToArchivedProjectException,
 )
 
+if TYPE_CHECKING:
+    from bravado.requests_client import RequestsResponseAdapter
+
 _logger = logging.getLogger(__name__)
 
 MAX_RETRY_TIME = 30
 retries_timeout = int(os.getenv(NEPTUNE_RETRIES_TIMEOUT_ENV, "60"))
 
 
+def ensure_json_body(response: "RequestsResponseAdapter") -> Dict[str, Any]:
+    try:
+        return response.json() or dict()
+    except Exception:
+        return {}
+
+
+def handle_json_errors(
+    response,
```

Comment:
Can we have a type hint for this argument? Plus a type hint for what the function returns?

Reply:
Sure. Done